app/resources/translations/lt_LT/validators.lt_LT.xlf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>Slaptažodžiai nesutampa.</source>
```

Comment:
<source> should not be translate, only <target>.
Original source is "Passwords must match." but you translated the opposite?
Please see https://crowdin.com/project/bolt

Reply:
@horus68 means it should be:
```
<source>Passwords must match.</source>
<target>Slaptažodžiai nesutampa.</target>
```
The `<source>` probably needs to be the original english text

Reply:
oh, ok, sorry, will modify it :)